.github/workflows/checks.yaml

```diff
@@ -0,0 +1,26 @@
+on: [push]
+name: Checks
+env:
+  DOCKER_IMAGE_GO: "golang:1.18.3@sha256:b203dc573d81da7b3176264bfa447bd7c10c9347689be40540381838d75eebef"
+  DOCKER_IMAGE_GOLINT: "golangci/golangci-lint:v1.45.2@sha256:e84b639c061c8888be91939c78dae9b1525359954e405ab0d9868a46861bd21b"
+  DOCKER_IMAGE_DOCKERLINT: "hadolint/hadolint:v2.9.2@sha256:d355bd7df747a0f124f3b5e7b21e9dafd0cb19732a276f901f0fdee243ec1f3b"
```

Comment:
What is the reason for overwriting those, since are hardcoded in the makefile?

Reply:
Great catch. Looks like i forgot those during testing :D let me remove them :D

Reply:
Those are removed